libs/@local/status/crate/rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
 channel = "nightly-2023-03-21"
-components = ['clippy', 'rustfmt']
+components = ['rustfmt', 'clippy']
```

Comment:
We're going for consistent ordering, I assume. What is the ordering based on? Personal preference, alphabetical ordering, or topical ordering?

Reply:
I was looking at all `rust-toolchain.toml` files anyway. It does not deserve additional documentation as otherwise, it would fall out of sync. This really does not matter at all.